src/modules/admin/wrappers/admin-core-config.tsx

```diff
@@ -54,7 +54,12 @@ const onSaveContentPage = async (contentPageInfo: ContentPageInfo) => {
 
 export function getAdminCoreConfig(router: NextRouter | null, locale: Locale): AdminConfig {
 	return {
-		staticPages: Object.values(ROUTES_BY_LOCALE.nl),
+		staticPages: Object.fromEntries(
+			Object.keys(ROUTES_BY_LOCALE).map((language) => [
+				language,
+				Object.values(ROUTES_BY_LOCALE[language as Locale]),
+			])
+		) as Record<Locale, string[]>,

```

Comment:
this is part of the admin-core config interface
so the less property names we encode in that admin-core-config type the better
since this config is shared with avo

But we can discuss it in person some time if you want. i'm open to both sides